.github/workflows/ci.yml

```diff
@@ -51,3 +51,19 @@ jobs:
       - name: Check (Linux)
         if: runner.os == 'Linux'
         run: ./gradlew $GRADLE_ARGS check
+
+      - name: PublishDocumentation
+        if: startsWith(github.ref, 'refs/tags/')
+        run: |
+          ./gradlew dokkaHtml
+          git config user.email CONX_GH_PAGES_USER_EMAIL
+          git config user.name CONX_GH_PAGES_USER_NAME
+          git fetch
+          git checkout gh-pages
+          rsync -a core/build/dokka/html/core/ .
```

Comment:
Probably don't need to copy, can just move, right?

Reply:
Was trying that first, but it was complaining because the [folder I was moving to] was not empty.

Reply:
I see.
Probably still more efficient to perform move than copy.
Something to the effect of:
```
mv core/build/dokka/html/core/ temp_core/
rm -rf core/
mv temp_core/ core/
```